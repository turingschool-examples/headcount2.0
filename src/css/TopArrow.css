.top-arrow-container {
  position: fixed;
  right: 0;
  bottom: 100px;
  width: 40px;
  height: 40px;
  margin: 0;
  border: 2px solid #000;
  background: #fff;
  border-right: 0;
  border-top-left-radius: 10px;
  border-bottom-left-radius: 10px;
}

.top-arrow-container:hover {
  cursor: pointer;
  background: #000;
}

.top-arrow-container:hover a img {
  transform: scale(1.1);
}

.top-arrow-container a {
}

.top-arrow-container a img {
  padding-left: 6px;
  padding-top: 5px;
  width: 30px;
  height: 30px;
  display: block;
  transition: .3s;
}

