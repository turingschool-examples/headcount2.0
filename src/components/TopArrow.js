import React from 'react';

import '../css/TopArrow.css';

const TopArrow = () => {
  return (
    <div 
      className="top-arrow-container"
      aria-label='top-of-page-link-wrapper'
    >
      <a 
        href="#page-top"
        aria-label='top-of-page-link'
      >
        <img
          src="./up-arrow.svg" 
          alt="up-arrow"
        />
      </a>
    </div>
  );
};

export default TopArrow;