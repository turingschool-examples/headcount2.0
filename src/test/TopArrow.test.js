import React from 'react';
import { shallow } from 'enzyme';
import TopArrow from '../components/TopArrow';


describe('TopArrow', ()=>{
  let wrapper; 

  beforeEach(()=>{
    wrapper = shallow(<TopArrow />);
  });

  it('should exist', () => {
    expect(wrapper).toBeDefined();
  });

  it('should render content to the DOM', () => {
    expect(wrapper).toMatchSnapshot();    
  });

});