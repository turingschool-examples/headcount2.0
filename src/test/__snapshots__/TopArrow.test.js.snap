// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopArrow should render content to the DOM 1`] = `
<div
  aria-label="top-of-page-link-wrapper"
  className="top-arrow-container"
>
  <a
    aria-label="top-of-page-link"
    href="#page-top"
  >
    <img
      alt="up-arrow"
      src="./up-arrow.svg"
    />
  </a>
</div>
`;
